src/components/Filter/Filter.tsx

```diff
@@ -1,45 +1,52 @@
+import { useOnClickOutside } from "usehooks-ts";
+import { useRef, useState } from "react";
+import { Content, Dropdown, Header, Item } from "./style";
+import dropdownIcon from "../../assets/icons/dropdown.svg";
 
-import { useState } from 'react';
-import { Content, Dropdown, Header, Item } from './style';
-import dropdownIcon from '../../assets/icons/dropdown.svg';
-
-export interface FilterProps{
-    options: string[],
-    name: string,
-    onChangeValue: (value: string) => void,
+export interface FilterProps {
+  options: string[];
+  name: string;
+  onChangeValue: (value: string) => void;
 }
 
-const Filter: React.FC<FilterProps> = (props: FilterProps)=> {
-    const[isActive,setIsActive] = useState(false);
-    const[selected,setSelected] = useState(props.name);
+const Filter: React.FC<FilterProps> = (props: FilterProps) => {
```

Comment:
Is not necessary to change here. only to make sure you know the syntax of destructing from props